internal/resources/ibm/ibm_cos_bucket.go

```diff
@@ -0,0 +1,44 @@
+package ibm
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// IbmCosBucket struct represents <TODO: cloud service short description>.
+//
+// <TODO: Add any important information about the resource and links to the
+// pricing pages or documentation that might be useful to developers in the future, e.g:>
+//
+// Resource information: https://cloud.ibm.com/<PATH/TO/RESOURCE>/
+// Pricing information: https://cloud.ibm.com/<PATH/TO/PRICING>/
+type IbmCosBucket struct {
+	Address string
+	Region  string
```

Comment:
region_location and storage_class should also be extracted as they will influence price. will also need to add capacity as a usage parameter.